src/canmatrix/formats/cmjson.py

```diff
@@ -193,11 +194,12 @@ def load(f, **_options):
                     size=signal["bit_length"],
                     is_little_endian=is_little_endian,
                     is_signed=is_signed,
-                    is_float=is_float,
-                    factor=signal["factor"],
-                    offset=signal["offset"]
+                    is_float=is_float
                 )
 
+                new_signal.factor = new_signal.float_factory(signal["factor"])
+                new_signal.offset = new_signal.float_factory(signal["offset"])
```

Comment:
Was `Signal` not properly applying the `float_factory` itself?

Reply:
Oh, I didn't catch that. Thanks